src/CommunityToolkit/Xamarin.CommunityToolkit/AttachedProperties/SetFocusOnEntryCompleted.shared.cs

```diff
@@ -36,12 +37,17 @@ public static void SetNextElement(BindableObject view, VisualElement value)
 
 		static void OnNextElementChanged(BindableObject bindable, object oldValue, object newValue)
 		{
-            var entry = bindable as Entry;
+			var entry = (Entry)bindable;
+			var weakEntry = new WeakReference<Entry>((Entry)bindable);
```

Comment:
```suggestion
			var entry = (Entry)bindable;
			var weakEntry = new WeakReference<Entry>(entry);
```
Could we use the previously casted variable?

Reply:
Oh, whoops, I didn't completely undo a change, thanks for noticing that.